qeneth

```diff
@@ -217,10 +217,10 @@ genyaml()
 					printf(" qn_name %s ", e.tailport);
 			}
 
-			if (e.qn_headmac && e.head == $)
-				printf(" qn_mac %s ", e.qn_headmac);
-			if (e.qn_tailmac && e.tail == $)
-				printf(" qn_mac %s ", e.qn_tailmac);
+			if (hasAttr(e, "qn_headmac") && e.qn_headmac && e.head == $)
+				printf(" qn_mac: %s ", e.qn_headmac);	
```

Comment:
Nitpick, extra whitespace introduced at the end of this line.

Reply:
thnx, fixed :)